mlflow/types/llm.py

```diff
@@ -305,6 +305,12 @@ def __post_init__(self):
                 f"Expected `usage` to be of type TokenUsageStats or dict, got {type(self.usage)}"
             )
 
+    def __setattr__(self, name, value):
+        # A hack to ensure users cannot overwrite 'object' field
+        if name == "object" and value != "chat.completion":
+            raise AttributeError("object must be 'chat.completion'")
```

Comment:
```suggestion
            raise ValueError("`object` field must be 'chat.completion'")
```

nit: In Python, `AttributeError` is typically used when accessing non-existing attribute. This is the case where the given value is invalid, so `ValueError` seems to be a better fit.

Reply:
ah got it. thanks for the explanation!